scripts/build-backend.sh

```diff
@@ -46,28 +46,38 @@ echo -en "Minifying with noderify..."
   --filter=async_hooks \
   --filter=utf-8-validate \
   --filter=bufferutil \
-  --filter=worker_threads \
```

Comment:
`noderify@5.1.1` filters this by default now: https://github.com/staltz/noderify/commit/ac8b095c56c045880fde5933ac1d02f88b20ef6a